skills/lookup.js

```diff
@@ -0,0 +1,55 @@
+const ExpressionList = require('../submodules/Regex/ExpressionList');
+const ping = require('ping');
+
+
+module.exports = controller => {
+    controller.hears(
+        [ExpressionList.lookup], 
+        'direct_message,direct_mention', 
+        async (bot, message) => {
+            bot.createConversation(message, async(err, convo) => {
+                if (err) return false;
+                let responseMessage;
+                let domain = message.match[1];
+
+                if(!domain){
+                    convo.say("Missing parameter. Please provide the domain name to lookup");
+                    convo.activate();
+                    return;
+                }
+                //Slack will automatically try to make anything ending in '.com' be a link
+                //This causes issue with message response formatting which needs to be corrected
+                if(domain[0] === '<'){ 
+                    // RegEx matches: <http:// and >
+                    var regex = /(<http:\/\/|>)/gi;
+                    domain = domain.replace(regex, '');
```

Comment:
Since the `regex` var is only used once, I'd just directly put the regex in the call that's using it:
`                    domain = domain.replace(/(<http:\/\/|>)/gi, '');`